airflow/providers/google/cloud/transfers/facebook_ads_to_gcs.py

```diff
@@ -149,7 +149,7 @@ def execute(self, context: Context):
             )
             raise AirflowException(message)
         total_row_count = self._decide_and_flush(converted_rows_with_action=converted_rows_with_action)
-        self.log.info("Facebook Returned %s data points in total: ", total_row_count)
+        self.log.info("Facebook Returned %s data points in total:", total_row_count)
```

Comment:
```suggestion
        self.log.info("Facebook Returned %s data points in total.", total_row_count)
```

I don’t even understand why the colon is there…